Dockerfile

```diff
@@ -1,17 +1,11 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-authentication-operator
 COPY . .
-RUN go build -tags="ocp" -o authentication-operator ./cmd/authentication-operator
+ENV GO_PACKAGE github.com/openshift/cluster-authentication-operator
+RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" -tags="ocp" -o authentication-operator ./cmd/authentication-operator
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.2:base
```

Comment:
v4.2, v4.1 do not exist, only v4.0 (other repos (KASop, OASop Dockerfile use the v4.0, also)